app/views/blocks/index.haml

```diff
@@ -3,7 +3,7 @@
 - content_for :flashes do
   .flash.error
     = image_tag "icons/exclamation.png".freeze, class: 'vmid'.freeze
-    Warning: blocking is not currently implemented. This page should remember any blocks you have created, but they will not take effect yet.
+    Warning: content blocking is not yet implemented, and will function the same as simply blocking posts
```

Comment:
We should clarify somewhere on this page if the post block setting isn't going to hide non-locked posts

Reply:
Yes, good point. Hmm... Words are hard.